modules/ROOT/pages/proc_knative-eventing.adoc

```diff
@@ -1,9 +1,33 @@
-= Installing the Knative Eventing Operator on OpenShift Container Platform
+//
+//
+= Installing the Knative Eventing Operator on Red Hat OpenShift Container Platform
 
-Knative Eventing can be installed by using the Knative Eventing Operator available on the OpenShift Container Platform web console.
+Knative Eventing can be installed by using the Knative Eventing Operator available on the Red Hat OpenShift Container Platform web console.
+
+For more information about using the Red Hat OpenShift Container Platform web console, see the link:https://docs.openshift.com/container-platform/4.2/web-console/web-console.html[Red Hat OpenShift Container Platform web console documentation].
 
 IMPORTANT: This release is a developer preview and should not be used in a production environment.
 
+During installation, the user must determine the following initial settings for the Operator:
+
+* Installation Mode
+* Update Channel
+* Approval strategy
+
+== Installation Mode
+
+* Choose **All namespaces on the cluster (default)** to have the Operator installed on all namespaces. You can also choose **A specific namespace on the cluster**, if available, to install the operator in a selected namespace.
+
+== Update Channel
+
+* If an Operator is available through multiple channels, you can choose the channel to where it is subscribed. For example, to deploy from the `example` channel, select it from the list drop-down list.
+
+== Approval Strategy
+
+* You can choose **Automatic** or **Manual** updates. If you choose **Automatic** updates for an installed Operator, when a new version of that Operator is available, the OLM automatically upgrades the running instance of your Operator.
+
+* If you select **Manual** updates, when a newer version of an Operator is available, the OLM creates an update request. As a cluster administrator, you must then manually approve that update request to have the Operator updated to the new version.
+
 IMPORTANT: With Knative 0.10.0 the Sources are now using the Destination API, and references to a `Sink` should be written as below. The old API still works, but was deprecated and will be removed in future releases.

```

Comment:
Should source and sink be capitalized?

...should be written as _described_ below